examples/react/basic/src/gen/eliza-BigIntService_connectquery.ts

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// @generated by protoc-gen-connect-query v0.5.3 with parameter "target=ts,import_extension=none,ts_nocheck=false"
+// @generated by protoc-gen-connect-query v0.5.1 with parameter "target=ts,import_extension=none,ts_nocheck=false"
```

Comment:
I'd expect `v0.6.0` here.

Reply:
Yeah without the release, we can't reference 0.6.0. I'll switch back to workspace to bring this back up to 0.6.0.